app/webpacker/components/RegistrationsV2/api/payment/get/refundPayment.js

```diff
@@ -7,7 +7,7 @@ export default async function refundPayment({
   amount,
 }) {
   return fetchWithAuthenticityToken(
-    refundPaymentUrl(competitionId, paymentId),
+    refundPaymentUrl(competitionId, 'stripe', paymentId),
```

Comment:
Looks like this is still hardcoded to Stripe - will we need to update it to also support Paypal in the future? I assume yes, and the reason we aren't doing so is because it's hard to pass a `paypal_enabled?` flag to the frontend?